discord/user.py

```diff
@@ -430,6 +430,14 @@ async def edit(self, **fields):
             Only applicable to user accounts.
         username: :class:`str`
             The new username you wish to change to.
+
+        .. note::
+
+            The new username cannot be ``discordtag``, ``everyone``, or ``here``.
+
+        .. warning ::
+            There is a strict ratelimit of changing your username of twice per 30 minutes.
+
```

Comment:
Indentation here is off. I'm not a fan of documenting rate limits since those can and have changed.

Reply:
Ah alright, that's a fair point. I'll remove that part and fix the indenting :)